Benchmark-Models/Lang_PLOSComputBiol2024/parameters_Lang_PLOSComputBiol2024.tsv

```diff
@@ -0,0 +1,376 @@
+kDpE2f2		log	7.34E-16	1.86E-11	8.20E-13	1	uniform	
+kSyEmi1		log	416982.6945	41699512.66	1317275.1	1	uniform	
+kSyEmi2		log	0.693000194	69.29966	4.023427865	1	uniform	
+kDeEmi1		log	3.53E-07	3.53E-05	1.64E-05	1	uniform	
+kDeEmi2		log	6.93E-07	6.93E-05	3.50E-05	1	uniform	
+kDeEmi3		log	0.003470002	0.347000173	0.005538575	1	uniform	
+kPhEmiA		log	5.75E-19	5.75E-17	8.97E-19	1	uniform	
+kPhEmiB		log	8.83E-15	2.88E-12	6.02E-14	1	uniform	
+kDpEmi		log	8.15E-09	2.89E-06	2.17E-06	1	uniform	
+kSyCa1		log	83499.84045	38511554.17	607075.3046	1	uniform	
+kSyCa2		log	0.104000456	62.13729164	0.225882008	1	uniform	
+kDeCa1		log	1.05E-10	1.05E-06	1.07E-10	1	uniform	
+kDeCa2		log	7.45E-14	7.45E-12	6.85E-13	1	uniform	
+kDeCa3		log	1.56E-20	7.50E-17	1.24E-17	1	uniform	
+kDpCdh		log	0.000346999	0.034699847	0.008890013	1	uniform	
+kPhCdhA		log	2.05E-16	3.03E-14	2.89E-14	1	uniform	
+kPhCdhE		log	1.70E-21	1.22E-17	8.29E-20	1	uniform	
+kPhCdhB		log	6.63E-21	6.63E-17	2.12E-20	1	uniform	
+kDiACE		log	5.96E-06	0.034699847	0.009196941	1	uniform	
+kAsACE		log	1.11E-13	5.75E-08	6.10E-10	1	uniform	
+kDiACdh		log	3.46E-07	3.47E-05	4.08E-07	1	uniform	
+kAsACdh		log	2.54E-14	2.33E-10	1.07E-10	1	uniform	
+kPhCeA		log	6.33E-20	2.62E-17	7.75E-20	1	uniform	
+kPhCeE		log	2.32E-21	2.38E-17	8.29E-20	1	uniform	
+kPhFoxE		log	1.15E-17	1.15E-13	4.99E-16	1	uniform	
+kPhFoxA		log	1.15E-16	9.95E-13	3.78E-15	1	uniform	
+kPhFoxB		log	5.75E-15	5.75E-13	5.21E-14	1	uniform	
+kDpFox		log	1.04E-08	1.04E-06	1.22E-08	1	uniform	
+kSyCb1		log	83499.84045	8350232.995	3336037.432	1	uniform	
+kSyCb2		log	0.104000456	10.40004362	9.843245212	1	uniform	
+kDeCb1		log	6.93E-08	6.93E-06	3.10E-06	1	uniform	
+kDeCb2		log	1.44E-16	1.44E-14	8.86E-15	1	uniform	
+kDeCb3		log	1.44E-15	1.44E-13	3.66E-15	1	uniform	
+kPhEnsa		log	1.15E-15	1.15E-13	4.38E-14	1	uniform	
+kDpEnsa		log	3.60E-05	0.347010583	0.003234241	1	uniform	
+kPhGw		log	1.14E-14	1.14E-12	9.22E-13	1	uniform	
+kDpGw1		log	0.000173	0.017299979	0.009615513	1	uniform	
+kDpGw2		log	1.15E-13	1.15E-11	6.67E-13	1	uniform	
+kWee1		log	1.11E-21	2.90E-18	1.68E-21	1	uniform	
+kWee2		log	1.15E-14	1.15E-10	1.30E-12	1	uniform	
+kPhWeeA		log	1.15E-15	4.97E-12	1.85E-14	1	uniform	
+kPhWeeB		log	3.22E-15	3.19E-11	4.61E-13	1	uniform	
+kDpWee		log	1.15E-13	1.15E-11	1.09E-11	1	uniform	
+kCdc25_1		log	1.15E-16	1.15E-14	8.65E-15	1	uniform	
+kCdc25_2		log	1.15E-14	1.15E-12	8.40E-14	1	uniform	
+kDpCdc25		log	1.15E-13	1.44E-09	5.13E-12	1	uniform	
+kDipEB55		log	4.71E-07	0.001383382	0.00067326	1	uniform	
+kAspEB55		log	1.70E-13	6.56E-10	1.54E-11	1	uniform	
+kSyCdc_1		log	1561.849756	8350232.995	260294.1445	1	uniform	
+kSyCdc_2		log	0.001237748	0.139	0.079613534	1	uniform	
+kDeCdc_1		log	2.77E-06	0.000277001	8.10E-06	1	uniform	
+kDeCdc_2		log	4.76E-21	4.76E-17	5.78E-21	1	uniform	
+kDipACdc		log	0.008660178	86.60356004	0.812910086	1	uniform	
+kAspACdc		log	1.17E-10	1.82E-07	1.06E-09	1	uniform	
+kPhApcA		log	3.19E-14	3.19E-12	2.60E-13	1	uniform	
+kDpApc_1		log	0	0.03	0	1	uniform	
```

Comment:
Hi @paulflang, the following parameters are set log-scale, but have `0` as `nominalValue`, is this correct? 
```
kDpApc_1
kDpE2f1
kPhC25A
kPhNup
kDpNup_1
kIm_2
kEx_2
```
Three out of those are marked as estimated. Shouldn't they be different from 0?

Reply:
Hmm. So the last four are set to zero and not estimated (I wanted to switch nuclear pore phosphorylation and unspecific nuclear import and export off).

For the first three, I always used zero as my guess (Note to myself: `kDpApc_1` and `kDpE2f1` are B55 independent background dephosphorylation rates. `kPhC25A` is CDC25 phosphorylation by CCNA. It is known that CCNB:CDK1 can phosphorylate CDC25, so I thought I let the optimizer figure out if CCNA:CDK1 can do it too). I just checked how David ran the optimization, and he just set the lower bound to -1e24 (in log space) and the optimal result for all three parameters was around -9.98e23. Bit strange that the optimizer did not move much from the lower bound in any of the cases.

Anyway, since the PEtab problem passes petablint, I think it is up to the optimizer how to handle zero lowerBound for log parameter scale. But I could also set lowerBound and nominalValue to a very low value, maybe 1e-100). @dweindl , what do you think?

Reply:
Makes sense. Thanks for the explanation, @paulflang. 

> since the PEtab problem passes petablint

There is the open [question](https://github.com/PEtab-dev/libpetab-python/pull/259) whether it should :).